server/app.py

```diff
@@ -232,4 +234,4 @@ def get_metrics():
 
 if __name__ == '__main__':
     logging.info(f"running on {args.host}, listening on port {args.port}")
-    uvicorn.run("app:app", host=args.host, port=args.port, reload=True)
+    uvicorn.run("app:app", host=args.host, port=args.port, reload=True)
```

Comment:
undo the change on this line, bring back the blank empty line under `uvicorn.run`